options/option.js

```diff
@@ -48,18 +48,25 @@
   });
 
   // status:pid
-  const oStatusPid = document.getElementsByClassName('option-status-pid');
-  for (let i = 0; i < oStatusPid.length; ++i) {
-    oStatusPid[i].addEventListener('change', (evt) => {
-      Config.save(
-        Constants.CONFIG_SHOW_STATUS_PID,
-        !!parseInt(evt.target.value)
-      );
-    });
-  }
-
-  Config.load(Constants.CONFIG_SHOW_STATUS_PID, (showPid) => {
-    oStatusPid[showPid ? 0 : 1].checked = true;
+  // status:title
+  const oStatusPid = document.getElementById('option-status-pid');
+  const oStatusTitle = document.getElementById('option-status-title');
+  oStatusPid.addEventListener('change', (evt) => {
+    console.log(evt.target.checked);
+    Config.save(Constants.CONFIG_SHOW_STATUS_PID, Boolean(oStatusPid.checked));
+  });
+  oStatusTitle.addEventListener('change', (evt) => {
+    console.log(evt.target.checked);
+    Config.save(
+      Constants.CONFIG_SHOW_STATUS_TITLE,
+      Boolean(oStatusTitle.checked)
+    );
+  });
+  Config.load(Constants.CONFIG_SHOW_PROBLEM_TIER, (show) => {
+    oStatusPid.checked = show;
+  });
+  Config.load(Constants.CONFIG_SHOW_PROBLEM_TIER_COLOR, (show) => {
+    oStatusTitle.checked = !(show === false); // default as true
```

Comment:
다른 상수를 사용하셨네요 ㅎㅎ 아래를 의도하신 게 맞죠?

그리고 사이트에서 문제 제목을 보여주는 게 기본값은 아니기 때문에, 문제 번호를 보여주는 걸 기본 값으로 하는 게 맞아보입니다 :)

```suggestion
  Config.load(Constants.CONFIG_SHOW_STATUS_PID, (isChecked) => {
    oStatusPid.checked = !(isChecked === false);
  });
  Config.load(Constants.CONFIG_SHOW_STATUS_TITLE, (isChecked) => {
    oStatusTitle.checked = Boolean(isChecked);
```

Reply:
done in b6aad92